tutorials/autotvm/tune_nnvm_arm.py

```diff
@@ -257,9 +262,17 @@ def tune_tasks(tasks,
     for i, tsk in enumerate(reversed(tasks)):
         prefix = "[Task %2d/%2d] " % (i+1, len(tasks))
 
+        # if we want to use spatial pack for depthwise convolution
```

Comment:
can we move this after the if block of `try_winograd`?

Reply:
like 'try_winograd' for loop?

Reply:
yes.